core/trino-parser/src/main/antlr4/io/trino/sql/parser/SqlBase.g4

```diff
@@ -213,6 +220,18 @@ propertyValue
     | expression    #nonDefaultPropertyValue
     ;
 
+catalogProperties
+    : '(' catalogPropertyAssignments ')'
+    ;
+
+catalogPropertyAssignments
+    : catalogProperty (',' catalogProperty)*
+    ;
+
+catalogProperty
+    : string EQ expression
+    ;
```

Comment:
This shouldn't be necessary. Use `properties`, instead, like all the other commands.